backend/domain.ml

```diff
@@ -0,0 +1,405 @@
+open Utils
+
+module Player = struct
+    type t = Zero | One
+    let (to_yojson, of_yojson) =
```

Comment:
YO KESKJA MENAN ?
Pourquoi "to YO json" ?

Reply:
yojson c'est la lib de json :p 
J'ai renommé à certain endroits où c'est possible par JSON, mais le nom des fonctions de conversion ça doit rester to_yojson et of_yojson